client/rest-high-level/src/main/java/org/elasticsearch/client/HighLevelRequests.java

```diff
@@ -87,53 +87,18 @@
 import java.util.HashMap;
 import java.util.Locale;
 import java.util.Map;
-import java.util.Objects;
 import java.util.StringJoiner;
 
-public final class Request {
```

Comment:
one more thing, not too convinced about naming, given that what these methods do is convert from high-level requests to low-level requests. maybe rather `Requests` or `RequestUtils`? not a big deal though.

Reply:
I'll rename it to `RequestConverters`. OK?

Reply:
++